src/tests/Microsoft.PowerFx.Interpreter.Tests/SerializationTests.cs

```diff
@@ -30,9 +30,26 @@ public void DateTimeSerializationTests()
                     var dateTimeValue = FormulaValue.New(dt);
                     var dateTimeValueDeserialized = (DateTimeValue)engine.Eval(dateTimeValue.ToExpression());
 
-                    Assert.Equal(dateTimeValue.Value, dateTimeValueDeserialized.Value);
```

Comment:
>Assert.Equal(dateTimeValue.Value, dateTimeValueDeserialized.Value); [](http://example.com/codeflow?start=20&length=67)

why are we removing this assert?
Removing this suggests that deserialization is losing something.